kong/resolver/access.lua

```diff
@@ -194,7 +194,8 @@ local function find_api(uri)
 end
 
 function _M.execute(conf)
-  local uri = ngx.var.uri
+  local uri = stringy.split(ngx.var.request_uri, "?")[1]
```

Comment:
Some plugins, like the Request Transformations, can update the querystring. Here we are calculating the base URI, and then later in `kong.lua` after all the "access" handlers in the plugins have been executed, we append the *possibly* updated querystring at the end. So here, since I just want the base URI, I remove the querystring.